bin/ghe-restore

```diff
@@ -345,6 +345,10 @@ if is_external_database_target_or_snapshot && $SKIP_MYSQL; then
 else
   echo "Restoring MySQL database from ${backup_snapshot_strategy} backup snapshot on an appliance configured for ${appliance_strategy} backups ..."
   ghe-restore-mysql "$GHE_HOSTNAME" 1>&3
+  # Clear GitHub Connect settings stored in the restored database
+  if ! $RESTORE_SETTINGS; then
+    ghe-ssh "$GHE_HOSTNAME" -- "/usr/local/share/enterprise/ghe-reset-gh-connect -y"
```

Comment:
How do we handle cross-version compatibility here? If the version of GHES doesn't have this script, will cause the overall restore to fail?

Reply:
Ooo good point. Yes it will. Fix incoming...

Reply:
Fixed in https://github.com/github/backup-utils/pull/770/commits/4d6ddb4089e7695e1f94d6c420dfee14f2950cfc

Reply:
Fixed betterer in https://github.com/github/backup-utils/pull/770/commits/f6c475096486aa1b9d226e389fc8d3e30983d753 😁